docs/rules/specify-peers-locally.md

```diff
@@ -0,0 +1,38 @@
+# specify-peers-locally
+
+💼 This rule is enabled in the following configs: ✔️ `legacy-recommended`, ✅ `recommended`.
+
+💡 This rule is manually fixable by [editor suggestions](https://eslint.org/docs/latest/use/core-concepts#rule-suggestions).
+
+<!-- end auto-generated rule header -->
+
+When you have a dependency declared in a package's `peerDependencies`, it's meant to be simply an expression of compatibility with that dependency.
+It also establishes a contract with consumers of your package that they should install a version of that dependency within the range that you've declared.
+Since you also need to use that package locally as part of development, it's generally a best practice to declare it in the package's `devDependencies` also, so that it's installed and available for use.
```

Comment:
Style nit: removing a second usage of the same word.

```suggestion
Since you also need to use that package locally as part of development, it's generally a best practice to declare it in the package's `devDependencies` as well, so that it's installed and available for use.
```

This might just be a "me" preference, I won't be offended at all if you ignore this 😄